src/librawspeed/decoders/SrwDecoder.cpp

```diff
@@ -52,21 +52,15 @@ RawImage SrwDecoder::decodeRawInternal() {
 
   if (32769 == compression)
   {
-    bool bit_order = false;  // Default guess
-    auto msb_hint = hints.find("msb_override");
-    if (msb_hint != hints.end())
-      bit_order = ("true" == (msb_hint->second));
+    bool bit_order = hints.get("msb_override", false);
     this->decodeUncompressed(raw, bit_order ? BitOrder_Jpeg : BitOrder_Plain);
     return mRaw;
   }
 
   if (32770 == compression)
   {
     if (!raw->hasEntry ((TiffTag)40976)) {
-      bool bit_order = (bits == 12);  // Default guess
-      auto msb_hint = hints.find("msb_override");
-      if (msb_hint != hints.end())
-        bit_order = ("true" == (msb_hint->second));
```

Comment:
Right now there are 3 strings used in the xml: 'true', 'false' and '' (empty). They are mapped to the same values as with the old code. It could be that the `>>` operator also maps 'True' to `true` which the current code does not. I'd say that would be an improvement, if anything.